### PR TITLE
feat: AI-1863-In-doc-editor-implement-auto-expandable-UI

### PR DESCRIPTION
**Pull request by gaurpulkit**
### Problem
https://altimateai.atlassian.net/browse/AI-1863

In doc editor panel, currently width of the doc editor is fixed. In small screens, some of the space in right side is unused. Fixing that.

### Solution

Changing the styles for doc editor

### Screenshot/Demo

A picture is worth a thousand words. Please highlight the changes if applicable.
Change:
Screenshot 2024-07-25 at 6 15 51 PM
Screenshot 2024-07-25 at 6 16 08 PM

Before:
Screenshot 2024-07-25 at 6 16 28 PM

### How to test

- run the extension with this branch
- open doc editor panel
- see if it fills the whole screen

## Checklist

- [x] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

**Review by mdesmet — APPROVED** [review]

File: webview_panels/src/modules/documentationEditor/styles.module.scss [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 
 .documentation-wrapper {
+  position: absolute;
```
Comment (saravmajestic) [inline webview_panels/src/modules/documentationEditor/styles.module.scss:5]: You will not need position absolute for this change. This element is already taking 100vw and vh

**Review by saravmajestic — CHANGES_REQUESTED** [review]
https://www.loom.com/share/6856f4614ffd426c8b2fe0a10c85a294?sid=87b2f1bc-9a94-4a70-9650-bc67e5b305e0

This section should limit the max width as we see in history tab, so it wont extend for very large screens

File: webview_panels/src/modules/documentationEditor/styles.module.scss (hunk 1 above)
Comment (gaurpulkit) [reply webview_panels/src/modules/documentationEditor/styles.module.scss:5]: `position: absolute` is required. Not doing that breaks the layout.
Screenshot 2024-07-26 at 12 19 51 PM

**Comment by gaurpulkit** [comment]
> https://www.loom.com/share/6856f4614ffd426c8b2fe0a10c85a294?sid=87b2f1bc-9a94-4a70-9650-bc67e5b305e0
> 
> This section should limit the max width as we see in history tab, so it wont extend for very large screens
Screenshot 2024-07-26 at 12 48 22 PM
added the change. confirm on your screen.

**Comment by gaurpulkit** [comment]
After sarav's changes:
Screenshot 2024-07-31 at 9 12 55 AM
Screenshot 2024-07-31 at 9 13 24 AM

**Review by saravmajestic — APPROVED** [review]

**Pull request merged**